test/binary/relocs.txt

```diff
@@ -17,4 +17,5 @@ section("reloc.name") {
 (;; STDOUT ;;;
 (module
   (type (;0;) (func (result i32))))
+XXXXX
```

Comment:
Exactly. I'm just using this to verify travis will fail on a failing test (which it doesn't right  now!)